sdk/apimanagement/api-management-custom-widgets-tools/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
let's remove this and use the one in the root of the mono-repo

Reply:
same with .prettierignore and .prettierrc.json

Reply:
ooh yes, I forgot to delete them, thanks!